File: LAB_1/EJERCICIOS/EJ1.cpp
#include <iostream>
using namespace std;

// Función que recibe un arreglo y su tamaño
int sumarArreglo(int arr[], int n) {
    int suma = 0;
    for (int i = 0; i < n; i++) {
        suma += arr[i];
    }
    return suma;
}

int main() {
    int n;
    cout << "Introduce la cantidad de números: ";
    cin >> n;

    int* numeros = new int[n]; // Arreglo dinámico

    cout << "Introduce los " << n << " números:" << endl;
    for (int i = 0; i < n; i++) {
        cin >> numeros[i];
    }

    cout << "La suma es: " << sumarArreglo(numeros, n) << endl;

    delete[] numeros; // Liberar memoria
    return 0;
}

File: LAB_1/EJERCICIOS/EJ1.py
def sumar_arreglo(arr):
    suma = 0
    for num in arr:
        suma += num
    return suma

# Pedir cantidad de números
n = int(input("Introduce la cantidad de números: "))

# Crear lista vacía y llenarla con datos del usuario
numeros = []
print(f"Introduce los {n} números:")
for i in range(n):
    num = int(input())
    numeros.append(num)

# Calcular y mostrar la suma
print("La suma es:", sumar_arreglo(numeros))
